python/gcs.py

```diff
@@ -63,22 +63,22 @@ def bitreader(f):
     Coroutine to read bits from a file-like.
     Use .send(N) to read N bits from the file.
     """
-    accum = 0
+    accumulator = 0
```

Comment:
> Il giorno 15/apr/2015, alle ore 21:13, Giovanni Bajo <notifications@github.com> ha scritto:
> 
> In python/gcs.py:
> 
> > @@ -63,22 +63,22 @@ def bitreader(f):
> >      Coroutine to read bits from a file-like.
> >      Use .send(N) to read N bits from the file.
> >      """
> > -    accum = 0
> > +    accumulator = 0
> 
> Why this name change?

Because is more clear!

-- 
 |/ Francesco Sacchi - Develer S.r.l.
-|  http://www.develer.com/ - http://www.bertos.org/
 |\ Tel: +39 055 3984627 ext 208